src/.vuepress/components/DeprecatedBadge.vue

```diff
@@ -1,5 +1,12 @@
 <template>
-  <Badge type="error" :text="`Deprecated since ${version}`"/>
+  <div>
+    <p>Beware, this SDK is deprecated.</p>
```

Comment:
Why this is inside the `DeprecatedBadge` component?

Reply:
I just changed this for my tests ; this page is not used.

Reply:
I just changed this for my tests ; this page is not used.

Reply:
But this component is used in other part of the documentation

Reply:
Ok, I've cancelled my modifications in this file.